### PR TITLE
Report the returned response body to log the error detail from cloud.redhat.com

### PR DESCRIPTION
**Pull request by martinkunc**
This should return (for log) the http body, for example if ingress returns http errors.

### CODE REVIEW

File: pkg/insights/insightsclient/insightsclient.go [hunk 1]
```diff
@@ -229,6 +225,17 @@ func (c *Client) Send(ctx context.Context, endpoint string, source Source) error
 	return nil
 }
 
+func responseBody(r *http.Response) string {
+	if r == nil {
+		return ""
+	}
+	body, _ := ioutil.ReadAll(r.Body)
```
Comment (tremes) [inline pkg/insights/insightsclient/insightsclient.go:232]: Is it OK to ignore the error here?

Comment (martinkunc) [reply pkg/insights/insightsclient/insightsclient.go:232]: I think so, see the original version. We are trying to log what we received from response. If we cannot, we will log empty string.

Comment (tremes) [reply pkg/insights/insightsclient/insightsclient.go:232]: Ahh yes...sorry I was looking rather at the green fields than the red ones.

**Pull request merged**